data/schema/filters/abilities.cfg

```diff
@@ -10,5 +10,10 @@
 	{SIMPLE_KEY affect_self s_bool}
 	{SIMPLE_KEY affect_allies s_bool}
 	{SIMPLE_KEY affect_enemies s_bool}
+	{SIMPLE_KEY value s_range_list}
+	{SIMPLE_KEY add s_range_list}
+	{SIMPLE_KEY sub s_range_list}
+	{SIMPLE_KEY multiply f_real}
+	{SIMPLE_KEY divide f_real}
```

Comment:
Adding extra stuff means more to review, and should mean more unit tests too.

Seems odd that `add` and `sub` support ranges, but `multiply` and `divide` don't.

Reply:
i reverted that